scripts/sSCP395_h5ad_CAS.qmd

```diff
@@ -0,0 +1,168 @@
+---
+title: "sSCP395 h5ad for CAS"
+format: html
+---
+
+
+```{python}
+import numpy as np
+import pandas as pd
+import scanpy as sc
+adata = sc.read_10x_h5("PBMChealthyNoSort3k2.0.0/pbmc_unsorted_3k_filtered_feature_bc_matrix.h5")
+adata
+```
+
+ ```{python}
+ meta = pd.read_csv('prep_files/orig_10xmultiome_metadata4h5ad.tsv', sep="\t", index_col=0)
+ adata.obs = meta
+ ```
+
+
+```{python}
+umap = pd.read_csv('PBMChealthyNoSort3k2.0.0/analysis/dimensionality_reduction/gex/umap_projection.csv', usecols=[1, 2]).to_numpy()
+tsne = pd.read_csv('PBMChealthyNoSort3k2.0.0/analysis/dimensionality_reduction/gex/tsne_projection.csv', usecols=[1, 2]).to_numpy()
+```
+
+
+```{python}
+adata.obsm["X_umap"] = umap
+adata.obsm["X_tsne"] = tsne
+```
+
+
+```{python}
+adata.write_h5ad("sSCP395.h5ad")
+```
+
+# start CAS notebook
+
+
+```{python}
+import scanpy as sc
+import warnings
+warnings.filterwarnings("ignore", category=FutureWarning)
+warnings.filterwarnings("ignore", category=UserWarning)
+sc.set_figure_params(dpi=80)
+```
+
+
+
+```{python}
+sc.pl.umap(adata, color='gex_cluster')
+```
+
+
+```{python}
+api_token = "dd3b0604-a839-4631-ad77-95342d159b63.3c697671-e650-404a-ae21-acf313d7d49c"
```

Comment:
Are we worried about this being exposed?  My sense is that it's less a security risk and more of a bookkeeping issue.  Is there a way to pass this into the quarto file instead?